pkg/term/term.go

```diff
@@ -86,6 +94,8 @@ func SaveState(fd uintptr) (*State, error) {
 	return &oldState, nil
 }
 
+// DisableEcho disable echo of the terminal connected to the given file descriptor
+// according to the given state
```

Comment:
"DisableEcho applies the specified state to the terminal connected to the file descriptor, with echo disabled"